site/integrations/aws-cloud-formation.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Launching a Stack with Scope to AWS
+menu_order: 20
+---
+
+The AWS CloudFormation template is the easiest way to get started with [Weave Net] and [Weave Scope]. CloudFormation templates provide developers and systems administrators a simple way to create a collection or a stack of related AWS resources, and it provisions and updates them in an orderly and predictable fashion.
+
+Use this specially created Weaveworks CloudFormation template to create an EC2 instance with all of the resources you need, including Weave Net and Weave Scope.
+
+Before launching the stack:
+
+* [Set up an Amazon Account](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/get-set-up-for-amazon-ec2.html)
+* [Create the Key Pairs]("http://docs.aws.amazon.com/gettingstarted/latest/wah/getting-started-prereq.html). You will need to reference the name of the key pairs when you create the stack.
+
+The link below will launch a sample app using a Cloudformation template, but you can swap out the IdentiOrca app and use your own app instead. 
+
+**Ready to launch a stack?  Click here to launch a stack to AWS:**
+
+![](cloudformation-launch-stack.png)[1]
```

Comment:
I couldn't figure out how get a link on an image with markdown.  Do you know how to do this?